src/components/Navbar/Nav.js

```diff
@@ -0,0 +1,24 @@
+import '../../styles/Nav.css';
+import React from 'react';
+import { Link } from 'react-router-dom';
+
```

Comment:
- [ ] 
I would strongly recommend you fix the live demo of the project, I suggest you use netlify to deploy your projects

Reply:
@alexander16108  thanks for the feedback.
The site is live on Netlify: https://math-magicians-microverse.netlify.app/